src/clj/puppetlabs/trapperkeeper/services/watcher/filesystem_watch_core.clj

```diff
@@ -81,42 +92,58 @@
    watcher :- (schema/protocol Watcher)]
   (let [dir-create? (fn [event]
                       (and (= :create (:type event))
-                           (fs/directory? (:path event))))]
+                           (fs/directory? (:full-path event))))]
     (DirWatchUtils/registerRecursive (:watch-service watcher)
                                      (->> events
                                           (filter dir-create?)
-                                          (map #(.toPath (:path %)))))))
+                                          (map #(.toPath (:full-path %)))))))
+
+(schema/defn get-event-maps-from-key :- [Event]
```

Comment:
I had it as that for a little bit locally. But I thought that the `stuff->things` pattern was only used when translating or coercing from one type to another. In that vein I could see renaming `clojurize` to `watch-event->event`, but the meaningful portion of this function seemed to call an accessor-like function on the passed in Java object. So I went with a java-y accessor-like name. Is the `stuff->things` naming convention often used for interop like that? I don't have much experience with it.